### ISSUE TITLE
Allow user to work in timezones besides UTC

### ISSUE THREAD

**Issue by thodson-usgs** [USGS-python/dataretrieval#3]
All data is converted to UTC; however, some users might prefer to work in a local timezone.

**Comment by jjkennedy** [comment]
Is it possible to only pull the date column from groundwater levels? I'm finding that some sites do not have a time associated with the measurement and it's throwing an error because it can't convert that to a datetime index.

**Comment by elbeejay** [comment]
To update the status of this issue:

- Some documentation on converting timezones out of UTC is provided in #62 based on a suggestion in #61
- Handling of "incomplete" date-time information, such as measurements without times, was handled in #58

At this time there is still no built-in functionality for getting queries back in whatever timezone you want, they do come back with UTC-localized timestamps.

**Comment by mishranurag** [comment]
I assume that the daily data is in local time zone. However, the label on the time has Z on it which can be confusing.

![image](https://user-images.githubusercontent.com/15083239/236643680-83da5c60-06ab-4874-b8eb-f1b9eea2abfa.png)

**Comment by wzell-usgs** [comment]
On the subject of UTC - Is it expected behavior that the returned datetime index is UTC +0 rather than local UTC?  E.g., for a randomly selected GW site with IV service, here is the dataRetrieval result (UTC+0):

![dataRetrieval](https://github.com/DOI-USGS/dataretrieval-python/assets/16402586/2016f318-9e27-41f6-a0c8-dd5193fd8d4a)

and here is the NWIS web interface 'view data' (UTC-6):

![webdata](https://github.com/DOI-USGS/dataretrieval-python/assets/16402586/1087e83b-2fb6-4001-ac53-5966474d317e)

I think can programmatically convert to local UTC using the tz_cd with the site information but perhaps it is preferred to have dataRetrieval handle that conversion?

**Comment by thodson-usgs** [comment]
Correct, we decided to stick with USGS, unlike the web interface. The reason being what is "local time" when you're working analyzing data from multiple timezones?

**Comment by thodson-usgs** [comment]
Closing as defunct. Two reasons:

1. The original target, the `nwis` module, is being decommissioned in favor of the new `waterdata` module.
2. The design decision to return UTC was confirmed in earlier discussion on this issue — defaulting to any 'local' timezone is ambiguous when analyzing data across multiple sites/regions. Users who need a different timezone can convert explicitly with `tz_convert()` after the call.

If timezone handling surfaces as a concern in the `waterdata` module specifically, please open a fresh issue so we can scope it to the new API.
